website/src/lib/oasst_api_client.ts

```diff
@@ -160,8 +160,8 @@ export class OasstApiClient {
   /**
    * Returns the valid labels for messages.
    */
-  async fetch_valid_text(): Promise<any> {
-    return this.get(`/api/v1/text_labels/valid_labels`);
+  async fetch_valid_text(messageId: string = null): Promise<any> {
+    return this.get(`/api/v1/text_labels/valid_labels${messageId && `?message_id=${messageId}`}`);
```

Comment:
You can pass an object as the 2nd params of the get method.

Reply:
Ah, much nicer